admin/server/auth/pkce.go

```diff
@@ -122,8 +128,136 @@ func (a *Authenticator) getAccessTokenForAuthorizationCode(w http.ResponseWriter
 		return
 	}
 
-	// Issue an access token
-	authToken, err := a.admin.IssueUserAuthToken(r.Context(), userID, authCode.ClientID, "", nil, nil)
+	// check if the client is a first-party auth client
+	isFirstParty := false
```

Comment:
Instead of hard-coding first party clients, maybe consider adding a column in the `auth_clients` table to indicate whether or not to issue a long-lived token or a refresh token? E.g. a dedicated column or a `scopes` column with a scope to indicate it.

Reply:
How would that field be populated? dynamic clients won't specify this and if we add more clients we will add to list of first party clients. I believe since we have refresh token support all clients should just use this. 

I don't think there is any standard `scope` to indicate this.

Reply:
Added `long_lived_access_token` scope for existing clients so that going forward all other clients get short lived tokens only.